docs/en/docs/benchmarks.md

```diff
@@ -1,6 +1,6 @@
 # Benchmarks
 
-Independent TechEmpower benchmarks show **FastAPI** applications running under Uvicorn as <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">one of the fastest Python frameworks available</a>, only below Starlette and Uvicorn themselves (used internally by FastAPI). (*)
+Independent TechEmpower benchmarks show **FastAPI** applications running among Uvicorn and Starlette (used internally by FastAPI) as <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">one of the fastest Python frameworks available</a>.
```

Comment:
Thanks! Why do you think "among" would make more sense than "under Uvicorn"?

Reply:
Hi, @tiangolo ! Thanks for the questions. 

We have talked about it in person at PyCon. The problem is that when you click on the link, FastAPI doesn't appear, and when it appears with a few clicks like [this](https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=cl&test=query&l=zijzen-7) it's above Starlette and Uvicorn. That's why I mentioned "among" instead of "under".
![image](https://github.com/tiangolo/fastapi/assets/5021565/57388a92-f385-4d0c-86ae-a008bf93e10b)